src/base/emu-i386/cpu.c

```diff
@@ -290,9 +302,14 @@ static void fpu_io_write(ioport_t port, Bit8u val, void *arg)
   case 0xf0:
     /* We need to check if the FPU exception is pending, and set IGNNE
      * if it is, before untriggering an IRQ. But we don't (and probably
-     * can't) emulate IGNNE properly. So the trick is to do fnclex in
+     * can't) always emulate IGNNE properly. So the trick is to do fnclex in
      * bios.S, then untrigger IRQ unconditionally. */
     pic_untrigger(13); /* done by default via int75 handler in bios.S */
+    /* DJGPP's int75 pm handler for example does not use fnclex;
+       but we can force it in return_from_hw_int() in dpmi.c, so
+       this only makes sense in DPMI, as this variable is not checked in RM */
```

Comment:
Not the comment I had
in mind. Since IGNNE is
something real that should
actually be emulated, I
wanted the full "excuse"
for not doing this for RM
just yet.

But whatever. :)

Reply:
well you can edit the comment in the GUI inline like you did in another patch. I don't mind really.

I have #IGNNE working properly (locally) with sim so this is going to need to change a bit soon anyway.